src/test/java/com/google/step/servlets/CommentServletTest.java

```diff
@@ -0,0 +1,235 @@
+package com.google.step.servlets;
+
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.step.datamanager.CommentManager;
+import com.google.step.model.Comment;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+@RunWith(JUnit4.class)
+public class CommentServletTest {
+
+  private static final long DEALID = 2;
+
+  private static final long ID_A = 1;
+  private static final long USERID_A = 3;
+  private static final String CONTENT_A = "Hello world";
+  private static final Comment COMMENT_A = new Comment(ID_A, DEALID, USERID_A, CONTENT_A);
+
+  private static final String UPDATE_CONTENT_A = "Update";
+  private static final Comment UPDATE_COMMENT_A =
+      new Comment(ID_A, DEALID, USERID_A, UPDATE_CONTENT_A);
+
+  private static final long ID_B = 2;
+  private static final long USERID_B = 4;
+  private static final String CONTENT_B = "Hello world2";
+  private static final Comment COMMENT_B = new Comment(ID_B, DEALID, USERID_B, CONTENT_B);
+
+  private CommentManager commentManager;
+
+  private CommentServlet commentServlet;
+
+  private CommentGetPostServlet commentGetPostServlet;
```

Comment:
Could you split the CommentGetPostServlet's tests into a separate file? Sometimes people might forget which file has the tests, so it's better to create a separate test class corresponding to each servlet.